analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -56,11 +57,19 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
             new InitializationVectorShouldBeRandom(),
             new HashesShouldHaveUnpredictableSalt());
 
-        protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules { get; } = ImmutableDictionary<DiagnosticDescriptor, RuleFactory>.Empty
+        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabled)
+        {
+        }
```

Comment:
Nitpick: These can be single-lined everywhere
```suggestion
        public SymbolicExecutionRunner() : this(AnalyzerConfiguration.AlwaysEnabled) { }
```